conf/pulsar_tools_env.sh

```diff
@@ -42,13 +42,13 @@
 # PULSAR_GLOBAL_ZK_CONF=
 
 # Extra options to be passed to the jvm
-PULSAR_MEM=${PULSAR_MEM:-"-Xmx128m -XX:MaxDirectMemorySize=128m"}
+PULSAR_TOOL_MEM=${PULSAR_MEM:-"-Xmx128m -XX:MaxDirectMemorySize=128m"}
```

Comment:
1. This will set PULSAR_TOOL_MEM to PULSAR_MEM. It will ignore PULSAR_TOOL_MEM if it is set in the environment.
2. In a proper override you do need to consider that legacy users will be surprised by this behavior so if PULSAR_TOOL_MEM is not set then check PULSAR_MEM

This same comment applies to PULSAR_TOOL_GC

Reply:
Done, thx